src/validations/ValidateOtp.js

```diff
@@ -0,0 +1,18 @@
+export default function ValidateOtp(values){
```

Comment:
try to add comments to main function ,just to tell other users what it is intended to do.